_pages/en/strategy-transform-high-performing-org.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: "Strategy to Transform IITB into a High Performing Organization (Continuous Improvement)"
+ref: transform-high-performing-org
+lang: en
+status: posted
+sections: Strategies
+permalink: /strategy-transform-high-performing-org.html
+---
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+  - [Governance, Compliance, and Reporting](#governance-compliance-and-reporting)
+  - [Finance](#finance)
+  - [Legal](#legal)
+  - [Architecture](#architecture)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+
+## Introduction
+
+### Purpose
+
+To provide IITB with a strategy to transform into a high performing organization through the continuous improvement of daily Work.
+The strategy includes:
+
+- A guiding policy, which serves to set automatic decisions that defines the continuous improvement strategy
+- A coherent set of actions (an action plan), which serve to operationalize the continuous improvement strategy
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
+
+This strategy document is targeted to stakeholders involved in determining how IITB can continue to meet the needs of the department (ESDC) and Canadians. More specifically ...  This includes both IITB and non-IITB stakeholders (such as from CFOB, IAERMB, and SSC). The list of stakeholders are listed in section Coherent set of actions and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
+
+The Guiding Policy, once operationalized, will target all ESDC personnel involved in IT projects, resources and investments.  All teams within IITB are expected to adhere to this policy.
+
+### Business Case
```

Comment:
This section doesn't seem fully aligned for the strategy, more towards Solution Delivery Model?

Missing something about continuous improvement, since it's a bit more tailored to deployments I guess?

Reply:
Maybe here we can strictly outline:
What are the overall benefits and outcomes
Value of the strategy
What objective should be focused 
What is the definition of success in our strategy (can we have a good idea about what success looks like and when we can expect it
Describe the approach to be used
etc